### PR TITLE
Make the bootstrap target unsandboxed

### PR DESCRIPTION
**Pull request by Tatskaari**

### CODE REVIEW

File: tools/please_go/BUILD [hunk 1]
```diff
@@ -26,6 +26,7 @@ genrule(
     ],
     outs = ["please_go.bootstrap"],
     binary = True,
+    sandbox = False,
```
Comment (peterebden) [inline tools/please_go/BUILD:29]: Doesn't this make it unsandboxed?

Comment (Tatskaari) [reply tools/please_go/BUILD:29]: Yeah, it's building it with `go build` to bootstrap so it can't be sandboxed.

Comment (peterebden) [reply tools/please_go/BUILD:29]: Right yeah that was my confusion. The sandboxed version does work at present though? Is there some specific case it doesn't?

Comment (Tatskaari) [reply tools/please_go/BUILD:29]: It doesn't work if you try and build this from a repo with sandboxing enabled.

**Review by peterebden — APPROVED** [review]
derp, of  course

**Pull request merged**